app/views/projects/samples/transfers/_transfer_modal.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= viral_dialog(open: open) do |dialog| %>
+  <%= dialog.with_header(title: t("projects.samples.transfers.new.title")) %>
+  <%= dialog.with_section do %>
+    <%= form_with(url: namespace_project_samples_transfer_path, method: :post) do |form| %>
+      <div
+        data-controller="projects--samples--transfers"
+        data-projects--samples--transfers-storage-key-value=<%=namespace_project_samples_url%>
+        data-projects--samples--transfers-target="field"
+        data-projects--samples--transfers-field-name-value="sample_ids[]"
+      />
+      <div class="grid gap-4">
+        <div class="form-field">
+          <%= form.label :new_project_id %>
```

Comment:
**nitpick:** I don't know where this comes from, sorry.  I think the label would be better reading `Project to transfer samples to` or something along those lines.  Thoughts @deepsidhu85 ?

Reply:
Looks like we need to add a translation for it. It just took `New Project` from the `new_project_id`

Reply:
Please see 2099343b875f556ba064a1ad636b1f3b4e670bdc.